src/Api/Orders.php

```diff
@@ -48,9 +48,9 @@ public function cancelAll(?string $market = null, ?bool $conditionalOrdersOnly =
         return $this->respond($this->http->delete(self::ORDERS_URI, null, compact('market', 'conditionalOrdersOnly', 'limitOrdersOnly')));
     }
 
-    public function history()
+    public function history(?string $market = null, ?int $start_time = null, ?int $end_time = null, ?int $limit = null)
```

Comment:
For the timestamps, can you accept a DateTimeInterface here, and transform them using the TransformsTimestamps trait?

See Markets@trades for an example.

Reply:
Hi, 
I did updates that you requested and pushed to my branch, do you need to do anything else to add it to this pull request? (I am not much experienced with git)

Reply:
All good!